packages/ckeditor5-indent/tests/indentblockcommand.js

```diff
@@ -132,6 +132,15 @@ describe( 'IndentBlockCommand', () => {
 					setData( model, '<paragraph blockIndent="indent-4">f[]oo</paragraph>' );
 					expect( command.isEnabled ).to.be.false;
 				} );
+
+				// Should be disabled for block items in Document Lists. See https://github.com/ckeditor/ckeditor5/issues/14155.
+				it( 'should be false for a block element inside a list item', () => {
+					model.schema.extend( 'paragraph', { allowAttributes: [ 'listItemId' ] } );
+
+					setData( model, '<parentBlock><paragraph listItemId="foo">[]bar</paragraph></parentBlock>' );
```

Comment:
```suggestion
					setData( model, <paragraph listItemId="foo">[]bar</paragraph>' );
```

Reply:
I removed this test because there are already tests for indentation that I already had to adjust in this PR (indentmulticommand.js).